src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from './reduxFeatures/store';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import './index.css';
+
+const container = document.getElementById('root');
+const root = createRoot(container);
+
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+    <Router>
+      <App />
+    </Router>
+    </Provider>
+  </React.StrictMode>
```

Comment:
You should provide a link to the Rails API (backend app) in the README file for the front end. do this and your project is good to go 👍🏾 💯